packages/evm/src/precompiles/11-bls12-map-fp2-to-g2.ts

```diff
@@ -47,6 +47,18 @@ export async function precompile11(opts: PrecompileInput): Promise<ExecResult> {
     if (opts._debug !== undefined) {
       opts._debug(`${pName} failed: ${e.message}`)
     }
+    // noble‑curves throws this for inputs that map to the point at infinity
+    if (e.message === 'bad point: ZERO') {
```

Comment:
This string is hardcoded, can we read this specific string from an export from BLS? Can a comment  be added here that this lacks the necessary test input? Can we somehow help here to generate this input? We know which error string we should hit, so can we somehow reverse-engineer the input to trigger this?

Reply:
From the chat with the testing team, I understood that they will publish updated test vectors that will include the zero point test for G1 and G2. We have reverse engineered the G1 scalar from the BigInt value that Paul used in noble-curves (which from I understand he obtained from someone at the EF), but he mentioned having no idea how to generate the equivalent scalar for G2. I would just wait for the testing team to share the test vectors (this doesn't have to be merged before if we feel strongly about this)